Nos/Service/FeatureFlags.swift

```diff
@@ -1,8 +1,8 @@
 import Foundation
 import Dependencies
 
-/// Enum to represent each feature flag. Add new feature flags here easily.
-enum FeatureFlag: String {
+///  Feature flags for enabling experimental or beta features.
```

Comment:
There's an extra space before the beginning of the comment.

Reply:
How did you catch that before SwiftLint? 😄